### PR TITLE
(PC-1697) use new label for type description in component and selectors

### PR DESCRIPTION
**Pull request by annemarie35**
Dépend de la PR API https://github.com/betagouv/pass-culture-api/pull/547

### CODE REVIEW

File: src/selectors/tests/createOfferer.spec.js [hunk 1]
```diff
@@ -4,27 +4,26 @@ import state from './mockState'
 describe('selectOffererById', () => {
   it('should select the global state', () => {
```
Comment (atusithi) [inline src/selectors/tests/createOfferer.spec.js]: on peut renommer le nom du test -> 'should retrieve offerer from state when id is given'

File: src/selectors/tests/createOfferer.spec.js [hunk 2]
```diff
@@ -4,27 +4,26 @@ import state from './mockState'
...
     const expected = {
```
Comment (atusithi) [inline src/selectors/tests/createOfferer.spec.js]: Séparer en // given // when // then
```
// given
const expected = ...

// when
const result = selectOffererById(state, offererId)

// then
expect(result).toEqual(result)
```

File: src/selectors/tests/createOfferer.spec.js (hunk 1 above)
Comment (annemarie35) [reply src/selectors/tests/createOfferer.spec.js]: 👍

File: src/selectors/tests/createOfferer.spec.js (hunk 2 above)
Comment (annemarie35) [reply src/selectors/tests/createOfferer.spec.js]: yes !

**Pull request merged**